ui-modules/blueprint-composer/app/components/providers/blueprint-service.provider.js

```diff
@@ -93,8 +93,8 @@ function BlueprintService($log, $q, $sce, paletteApi, iconGenerator, dslService,
                     source: entity,
                     target: relation.entity,
                     label: relation.name,
-                    pathSelector: 'relation', // the CSS class for path
-                    labelSelector: 'relation-label', // the CSS class for label
+                    pathSelector: 'relation-selector', // the CSS class for path
+                    labelSelector: 'relation-selector', // the CSS class for label
```

Comment:
`pathSelector` and `labelSelector` both  seem to have the same value. Is this intended?

Reply:
Nevermind, I noticed there is the only one css class for this purpose.

Reply:
Yes, selectors are the same for arcs and labels for Brooklyn relationships, however, they are can be different in the downstream versions when selecting label and arc can be managed differently.